### PR TITLE
Fix lidarr.json

### PR DESCRIPTION
**Pull request by pigfrown**
Fixes lidarr plugin admin page not being available

### CODE REVIEW

File: lidarr.json [hunk 1]
```diff
@@ -14,5 +14,11 @@
     "pkgs": [
         "lidarr"
     ],
+    "properties": {
+        "allow_mlock": 1,
+        "nat": 1,
+        "nat_forwards": "tcp(8686:8686)"
+    },
+
```
Comment (sonicaj) [inline lidarr.json:22]: Why an empty line ?

**Review by sonicaj — COMMENTED** [review]
Thank you @pigfrown for your contribution!

File: lidarr.json (hunk 1 above)
Comment (pigfrown) [reply lidarr.json]: fixed

**Review by sonicaj — APPROVED** [review]
Thanks!

**Pull request merged**